server/test/routes/users.test.js

```diff
@@ -56,8 +56,8 @@ describe('Testing user routes', () => {
         .end((err, res) => {
           expect(res.status).to.equal(201);
           expect(res.body).to.haveOwnProperty('message')
-            .to.equal(`Sign up successful, visit your email 
-          to verify your account.`);
+            .to.equal(
+              `Sign up successful, visit your email to verify your account.`);
```

Comment:
did you write this test ? if so why the change ? pls only touch what's related to your current task